alica_engine/include/engine/collections/RobotProperties.h

```diff
@@ -17,8 +17,8 @@ struct AgentAnnouncement;
 class RobotProperties
 {
 public:
-    RobotProperties(const PlanRepository& planRepository, const std::string& defaultRole, const AgentAnnouncement& aa);
-    ~RobotProperties();
+    RobotProperties(const std::string& defaultRole, const AgentAnnouncement& aa);
+    ~RobotProperties() = default;
```

Comment:
Remove entirely?

Reply:
it was empty in cpp file

Reply:
yeah, but doesn't seem to be needed currently

Reply:
I guess all this class is redundant